requirements.txt

```diff
@@ -13,20 +13,20 @@ ipython==7.22.0
 jinja2==3.0.1
 jsonschema==3.2.0
 jwedb>=0.0.6
-jwst==1.2.3
+jwst==1.3.0
 matplotlib==3.4.2
 nodejs==10.13.0
-numpy==1.21.0
+numpy==1.18.5
```

Comment:
It looks like the `conda` environments are able to support `numpy==1.21.1`, so I think this can be updated as such here(?)